bindings/python/unicorn/unicorn.py

```diff
@@ -223,6 +223,9 @@ class uc_x86_xmm(ctypes.Structure):
         ("high_qword", ctypes.c_uint64),
     ]
 
+class uc_arm64_neon128(uc_x86_xmm):
```

Comment:
because this struct is independent from uc_x86_xmm, do you want to create a new class, without reusing uc_x86_xmm? even if the content duplicates uc_x86_xmm, it is fine